test/test_jit_fuser_te.py

```diff
@@ -1255,6 +1255,7 @@ def apply(fn):
             F.hardtanh,
             F.hardsigmoid,
             F.hardswish,
+            F.softplus,
```

Comment:
Based on test failures, it looks like this should also be added to `works_list` lower in this file

Reply:
Added later.